Updating the readme for the example template (#26)
dwave-examples/template: README.md

python <demo_name>.py
```

### Inputs
If your example requires user input, make sure to specify the input format and any input limitations.
### Outputs
An example program output.

## Problem Description 

Objectives to be Optimized: the goals the process attempts to accomplish by minimizing or maximizing certain aspects of the problem to the extent possible.
* Minimize the time to produce all of the products

Constraints: aspects of the problem and/or process with limited or no flexibility, which must be satisfied for solutions to be considered feasible.
* Machine A can only bend 10 parts per hour

## Model Overview

### Parameters
### Variables

> ### Expressions

Reviewer: ```suggestion
### Expressions
List and define any combinations of variables used for easier representations of the models.

```

### Objective

> Matching the objective description in the previous section with the mathematical formulation of the expression based on the problem paramaters and variables

Reviewer: ```suggestion
Mathematical formulation of the objective described in the previous section using the listed parameters, variables, etc.

```

### Constrains

> Matching the constraint description(s) in the previous section with the mathematical formulation of the expression based on the problem paramaters and variables

Reviewer: ```suggestion
Mathematical formulation of the constraints described in the previous section using the listed parameters, variables, etc.
```

python <demo_name>.py
```

> If your example requires user input, make sure to specify any input limitations.

Reviewer: Something to consider: now that people can come directly to one of our OS examples in GitHub and the repo provides the ability to just click it open in CodeSpaces (rather than the Leap IDE), the above Usage section should probably start with a line saying that a Leap account is needed.
